Source/Mosa.Korlib/System/String.cs

```diff
@@ -885,5 +885,64 @@ private string CreateTrimmedString(int start, int end)
 				return string.Ctor(this, start, len);
 			}
 		}
+
+		public unsafe static string FromPointer(byte* p, int size)
+		{
+			string s = "";
+			for (int i = 0; i < size; i++)
+			{
+				s += ((char)p[i]).ToString();
+			}
+			return s;
+		}
+
+		public unsafe static string FromPointer(byte* p, int size,byte ignore)
+		{
+			string s = "";
+			for (int i = 0; i < size; i++)
+			{
+				if ((char)p[i] == ignore) continue;
+				s += ((char)p[i]).ToString();
```

Comment:
Also, this creates lots of strings... try to preallocate the space first. See the previous comment.

Reply:
Sure!